hack/deployer/config/plans.yml

```diff
@@ -76,6 +80,30 @@ plans:
     region: europe-west1
     nodeCount: 3
     overwriteDefaultKubeconfig: false
+- id: eks-ci
+  operation: create
+  clusterName: ci
+  provider: eks
+  machineType: c5d.2xlarge
+  serviceAccount: false
+  kubernetesVersion: 1.14
+  eks:
+    region: eu-central-1
+    nodeCount: 3
+    nodeAMI: static
+    diskSetup: kubectl apply -f hack/deployer/config/local-disks/ssd-provisioner.yaml
```

Comment:
This makes it mandatory to run the deployer from the root path.
It doesn't really bother me but I got caught. I'm used to using it from `hack/deployer` to avoid specifying plans and configuration files via the flags.

FWIW, my workflow is:
```sh
cd hack/deployer
go build -o deployer main.go
./deployer execute
```